opcodes.json

```diff
@@ -272,7 +272,9 @@
     "input": "address",
     "output": "balance",
     "description": "Get balance of the given account",
-    "dynamicFee": true
+    "dynamicFee": {
+      "since": "berlin"
```

Comment:
@peroket wit this commit, you can add `since` (inclusive) and the dynamic part will only be visible if the selected fork is past the since